docs/scripts/convert-notebooks.py

```diff
@@ -44,13 +47,59 @@ def preprocess_cell(self, cell, resources, cell_index):
 )
 
 
+def add_frontmatter(
+    body: str,
+    nb_path: Path,
+) -> str:
+    frontmatter = {"custom_edit_url": NOTEBOOK_BASE_EDIT_URL + str(nb_path)}
+    formatted_frontmatter = "\n".join(f"{key}: {value}" for key, value in frontmatter.items())
```

Comment:
can we use `yaml.dump`?

```suggestion
    formatted_frontmatter = yaml.dump(frontmatter)
```